src/popups/HandicapPopup/HandicapPopupElement.test.js

```diff
@@ -3,6 +3,7 @@ import { configure, mount } from 'enzyme';
 import Adapter from 'enzyme-adapter-react-16';
 import renderer from 'react-test-renderer';
 import HandicapPopupElement from './HandicapPopupElement';
+import '../../utils/polyfills/flat';
```

Comment:
Can't you add react-app-polyfill/ie11'  and 'react-app-polyfill/stable' instead ?

Reply:
Even better you are right :)